platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/util/HashLogger.java

```diff
@@ -43,39 +41,24 @@
 public class HashLogger {
```

Comment:
The existing javadoc is not fully relevant when taking your changes into account. This component no longer queues anything, nor does it have a background thread.

Reply:
@kfa-aguda Your most recent commit undid the fix you made here